packages/react/src/components/form/Radio/Radio.tsx

```diff
@@ -1,28 +1,37 @@
 import type { ReactNode } from 'react';
 import { forwardRef } from 'react';
 
+import type { DefaultProps } from 'packages/react/src/types';
+import type { Merge } from 'packages/react/src/utilities';
```

Comment:
This should should be relative paths?

Reply:
This was auto-suggested by VS Code due to `baseUrl` in our root `tsconfig.json`.
<img width="750" alt="image" src="https://github.com/user-attachments/assets/5854ff2c-3442-425b-acbd-ff682454a090">

Good think you point this out, though, because the React build doesn't resolve paths from tsconfig, and thus output non-working paths in `dist`. We should either
1. add [rollup-plugin-tsconfig-paths](https://www.npmjs.com/package/rollup-plugin-tsconfig-paths) to our Rollup config (or better yet, Vite + vite-tsconfig-paths 🤓)
2. or not define any `baseUrl` / `paths` in a tsconfig which is reachable from the React package

Reply:
Follow up: [rollup-plugin-tsconfig-paths](https://www.npmjs.com/package/rollup-plugin-tsconfig-paths) doesn't change the .d.ts files output by `tsc`, of course, so it would require a different solution. I've used [typescript-transform-paths](https://www.npmjs.com/package/typescript-transform-paths) before, which is a TypeScript plugin. However, in this case I think we should just prevent these imports from happening. This works:

```json
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "@doc-components*": [
        "apps/storybook/docs-components",
        "apps/storybook/docs-components/*"
      ],
      "@assets/*": ["apps/storybook/assets/*"],
      // Prevent imports from e.g. 'packages/react', which will not work for consumers
      "packages/*": ["this/dir/does/not/exist/*"] // <------ added this line
    },
```

<img width="573" alt="image" src="https://github.com/user-attachments/assets/8dcb95bf-8309-42e5-acbc-64e64f862610">